src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesManagement.Codeunit.al

```diff
@@ -50,7 +50,7 @@ codeunit 396 NoSeriesManagement
         UnincrementableStringErr: Label 'The value in the %1 field must have a number so that we can assign the next number in the series.', Comment = '%1 = New Field Name';
 
 #if not CLEAN24
-    [Obsolete('Please use the "No. Series" and "No. Series - Batch" codeunits instead', '24.0')]
+    [Obsolete('Please use procedure TestManual(NoSeriesCode: Code[20]) in codeunit "No. Series" instead.', '24.0')]
```

Comment:
Gert had another Pr changing `procedure`/`function` to `method` in the obsolete description. Are we following that?